ee/codegen/src/generators/workflow-value-descriptor-reference/BaseNodeInputWorkflowReferencePointer.ts

```diff
@@ -0,0 +1,50 @@
+import { AstNode } from "@fern-api/python-ast/core/AstNode";
+import { Writer } from "@fern-api/python-ast/core/Writer";
+
+import { WorkflowContext } from "src/context";
+import {
+  IterableConfig,
+  WorkflowValueDescriptorReference as WorkflowValueDescriptorReferenceType,
+} from "src/types/vellum";
+
+export declare namespace BaseNodeInputWorkflowReferencePointer {
+  export interface Args<T extends WorkflowValueDescriptorReferenceType> {
+    workflowContext: WorkflowContext;
+    nodeInputWorkflowReferencePointer: T;
+    iterableConfig?: IterableConfig;
+  }
+}
+
+export abstract class BaseNodeInputWorkflowReferencePointer<
```

Comment:
This is neat

Reply:
[nit] Reference and Pointer are a bit redundant with each other, Id go with the former